e2e/suites/management/utils.go

```diff
@@ -291,15 +319,48 @@ func createSchedulerWithRoomsAndWaitForIt(t *testing.T, maestro *maestro.Maestro
 			"$ROOMS_API_ADDRESS/scheduler/$MAESTRO_SCHEDULER_NAME/rooms/$MAESTRO_ROOM_ID/ping " +
 			"--data-raw '{\"status\": \"ready\",\"timestamp\": \"12312312313\"}' && sleep 1; done"},
 	)
+	require.NoError(t, err)
+
+	// Change RoomsReplicas
 
-	// Add rooms to the created scheduler
-	// TODO(guilhermecarvalho): when autoscaling is implemented, this part of the test can be deleted
-	addRoomsRequest := &maestroApiV1.AddRoomsRequest{SchedulerName: scheduler.Name, Amount: 2}
-	addRoomsResponse := &maestroApiV1.AddRoomsResponse{}
-	err = managementApiClient.Do("POST", fmt.Sprintf("/schedulers/%s/add-rooms", scheduler.Name), addRoomsRequest, addRoomsResponse)
+	roomsReplicas := int32(2)
+	patchSchedulerRequest := &maestroApiV1.PatchSchedulerRequest{RoomsReplicas: &roomsReplicas}
+	patchSchedulerResponse := &maestroApiV1.PatchSchedulerResponse{}
+	err = managementApiClient.Do("PATCH", fmt.Sprintf("/schedulers/%s", scheduler.Name), patchSchedulerRequest, patchSchedulerResponse)
```

Comment:
Now I understand why the schedulers are starting at v1.1.0 🌝

Reply:
hehe

Reply:
they are still staring at v1.0.0 👀